docs/archive.md

```diff
@@ -0,0 +1,26 @@
+# Data Archiving
+
+When the crawls are complete, the data is archived to several places:
+
+- Internet Archive (sql)
+- HTTP Archive (sql, csv)
+- Google Storage (har)
+- BigQuery (har)
+
```

Comment:
Yeah, I guess that's fine. We can hopefully kill off the sql/csv bits very soon (tm).